Makefile

```diff
@@ -58,6 +63,14 @@ subsystem: build-image
 generate:
 	go generate $(shell go list ./...)
 
+generate-from-swagger: clean
```

Comment:
Need to have a way to get specific version of swagger - in order to get a swagger that is not in the master branch

Reply:
Currently it was done by go get github.com/filanov/bm-inventory@<hash>

Reply:
In addition, when developing in this repo, I prefer to decide what version of swagger (and generated files) I am working on without having to specify them every time I build it.

Reply:
The version of the bm-inventory client is decided the same way and can be changed with the same command
```bash
github.com/filanov/bm-inventory@git-hash
```
The only difference is now the generated files won't be a part of the bm-inventory package, but generated by each client by itself.

Reply:
> In addition, when developing in this repo, I prefer to decide what version of swagger (and generated files) I am working on without having to specify them every time I build it.

@eranco74 That's the opposite of what you wanted to achieve with one script.

Reply:
OK